code/game/objects/structures/stool_bed_chair_nest/chairs.dm

```diff
@@ -18,12 +18,6 @@
 	var/propelled = FALSE // Check for fire-extinguisher-driven chairs
 	var/comfort = 0
 
-/obj/structure/chair/New()
-	..()
-	spawn(3)	//sorry. i don't think there's a better way to do this.
-		handle_rotation()
```

Comment:
This proc is meant to handle rotating mobs buckled to a chair when the chair spawns and swaps direction. I cannot think of a single instance where theres a chair being spawned at mapload with a mob somehow pre-buckled that needs to be set in a specific direction.